ui/app/actions.js

```diff
@@ -348,9 +348,25 @@ function navigateToNewAccountScreen () {
   }
 }
 
-function addNewAccount () {
+function addNewAccount (oldIdentities) {
```

Comment:
we can get this using `getState` instead of relying on caller of the action to pass it in

Reply:
Done in 69682a3f